src/main/java/pl/byczazagroda/trackexpensesappbackend/controller/WalletRestController.java

```diff
@@ -0,0 +1,35 @@
+package pl.byczazagroda.trackexpensesappbackend.controller;
+
+import lombok.NonNull;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.CreateWalletDto;
+import pl.byczazagroda.trackexpensesappbackend.model.dto.WalletDto;
+import pl.byczazagroda.trackexpensesappbackend.service.WalletServiceImpl;
+
+import javax.validation.Valid;
+
+import static org.springframework.http.HttpStatus.CREATED;
+@RestController
+@RequiredArgsConstructor
+@Slf4j
+class WalletRestController {
+
+  @NonNull
+  private final WalletServiceImpl walletService;
+
+  @PostMapping("api/v1/wallets")
+  public ResponseEntity<WalletDto> create(@Valid @RequestBody CreateWalletDto createWalletDto) {
```

Comment:
The task was going on create empty class with `@RestController` annotation

Reply:
No, it was not specified in the task.
![image](https://user-images.githubusercontent.com/30751691/189069533-65f67ad9-53c0-429f-bc56-d5183cc8e7e4.png)

Reply:
Whats the point of making empty class?

Reply:
The point, is the task was about creating app core/structure